projects/demo/src/modules/directives/header/index.html

```diff
@@ -0,0 +1,36 @@
+<tui-doc-page
+    header="Header"
+    package="LAYOUT"
+    type="directives"
+>
+    <ng-template pageTab>
+        <tui-doc-example
+            id="base"
+            description="Buttons/badges sizes according to the header size"
+            heading="Sizes"
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample: 'html'"
+        />
+
+        <tui-doc-example
+            id="accessories"
+            heading="Accessories"
+            [component]="2 | tuiComponent"
+            [content]="2 | tuiExample: 'html,less'"
+        />
+
+        <tui-doc-example
+            id="interactive"
+            heading="Interactive"
+            [component]="3 | tuiComponent"
+            [content]="3 | tuiExample: 'html,less'"
+        />
+    </ng-template>
+
+    <ng-template pageTab="Setup">
+        <tui-setup
```

Comment:
We can probably simplify it even further by dropping `ng-template` and writing:
```html
<tui-setup
  *pageTab="'Setup'"
  ...
```

Reply:
great point 👍